README.md

```diff
@@ -163,6 +163,57 @@ diktat {
 
 You can run diktat checks using task `diktatCheck` and automatically fix errors with tasks `diktatFix`.
 
+## Run with Spotless
+[Spotless](https://github.com/diffplug/spotless) is a linter aggregator.
```

Comment:
```suggestion
[Spotless](https://github.com/diffplug/spotless) is a linter aggregator. Diktat can be run via spotless-gradle-pugin since version 5.10
```